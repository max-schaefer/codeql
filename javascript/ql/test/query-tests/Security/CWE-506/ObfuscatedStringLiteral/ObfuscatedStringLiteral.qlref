Security/CWE-506/ObfuscatedStringLiteral.ql
