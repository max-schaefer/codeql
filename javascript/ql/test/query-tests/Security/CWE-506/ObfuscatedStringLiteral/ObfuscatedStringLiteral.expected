| fast-request.js:1:16:1:57 | "\\x64\\x ... 6A\\x73" | This string literal looks deliberately obfuscated (its value is 'discord.js'). |
| fast-request.js:1:60:1:89 | "\\x72\\x ... 73\\x74" | This string literal looks deliberately obfuscated (its value is 'request'). |
| fast-request.js:1:104:1:125 | "\\x63\\x ... 63\\x68" | This string literal looks deliberately obfuscated (its value is 'catch'). |
| fast-request.js:1:128:1:345 | "\\x68\\x ... 67\\x3D" | This string literal looks deliberately obfuscated (its value is 'https://ennakuvis0tpi.x.pipedream.net/index.php?debug='). |
| fast-request.js:1:392:1:413 | "\\x6C\\x ... 69\\x6E" | This string literal looks deliberately obfuscated (its value is 'login'). |
